src/pykeen/evaluation/rank_based_evaluator.py

```diff
@@ -595,3 +595,43 @@ def finalize(self) -> RankBasedMetricResults:  # noqa: D102
             adjusted_arithmetic_mean_rank_index=dict(asr[ADJUSTED_ARITHMETIC_MEAN_RANK_INDEX]),
             hits_at_k=dict(hits_at_k),
         )
+
+
+def expected_mean_rank(
```

Comment:
maybe leave a note about why expected MRR is left out (because we tried and the math is hard!!)

Reply:
edf9698